app/lib/features/news/widgets/news_item_slide/text_slide.dart

```diff
@@ -4,42 +4,91 @@ import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
-class TextSlide extends ConsumerWidget {
+class TextSlide extends ConsumerStatefulWidget {
   final NewsSlide slide;
   final Color bgColor;
   final Color fgColor;
+  final PageController pageController;
 
   const TextSlide({
     super.key,
     required this.slide,
     required this.bgColor,
     required this.fgColor,
+    required this.pageController,
   });
 
   @override
-  Widget build(BuildContext context, WidgetRef ref) {
-    final slideContent = slide.msgContent();
+  ConsumerState<TextSlide> createState() => _TextSlideState();
+}
+
+class _TextSlideState extends ConsumerState<TextSlide> {
+  late ScrollController _scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController = ScrollController();
+    _scrollController.addListener(_onScroll);
+  }
+
+  void _onScroll() {
+    final topPosition = _scrollController.position.pixels <= 0;
+    final outOfRange = _scrollController.position.outOfRange;
+    final offset = _scrollController.offset;
+    final maxScrollExtent = _scrollController.position.maxScrollExtent;
+    if (topPosition && outOfRange && offset < -150) {
+      widget.pageController.previousPage(
+        duration: const Duration(milliseconds: 300),
+        curve: Curves.ease,
+      );
+    } else if (!topPosition && outOfRange && offset > maxScrollExtent + 150) {
+      widget.pageController.nextPage(
+        duration: const Duration(milliseconds: 300),
+        curve: Curves.ease,
+      );
+    }
+  }
+
+  @override
+  void dispose() {
+    _scrollController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final slideContent = widget.slide.msgContent();
     final formattedText = slideContent.formattedBody();
+
     return Container(
-      padding: const EdgeInsets.all(20),
-      color: bgColor,
+      padding: const EdgeInsets.symmetric(horizontal: 26,vertical: 70),
```

Comment:
```suggestion
      padding: const EdgeInsets.symmetric(horizontal: 26, vertical: 70),
```

Reply:
Which value is changed in suggestion?

Reply:
no value, just a space after the comma for style.